easyfix-messages/easyfix-messages-gen/src/gen.rs

```diff
@@ -387,33 +385,19 @@ impl Generator {
 
             impl fmt::Display for FieldTag {
                 fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-                    write!(f, "{}", self.as_fix_str())
+                    match self {
+                        #(FieldTag::#fields_names => write!(f, #fields_names_as_str),)*
+                    }
                 }
             }
 
             impl FieldTag {
-                pub const fn from_tag_num(tag_num: TagNum) -> Option<FieldTag> {
+                fn from_tag_num(tag_num: TagNum) -> Option<FieldTag> {
                     match tag_num {
                         #(#fields_numbers_literals => Some(FieldTag::#fields_names),)*
                         _ => None,
                     }
                 }
-
-                pub const fn as_bytes(&self) -> &'static [u8] {
```

Comment:
Do you see any technical reason to remove this?

At least one project uses this functions.